ThreeBotPackages/zerobot/admin/actors/admin.py

```diff
@@ -56,7 +55,15 @@ def set_explorer(self, explorer_type, schema_out=None, user_session=None):
         set current explorer (testnet/main)
         """
         if explorer_type in explorers:
-            j.core.myenv.config["EXPLORER_ADDR"] = explorers[explorer_type]
-            j.core.myenv.config_save()
+            client = j.clients.explorer.get(name=explorer_type, url=f"https://{explorers[explorer_type]}/explorer")
+            # check if we can switch with existing identity
+            try:
+                user = client.users.get(name=j.me.tname, email=j.me.email)
+            except j.exceptions.NotFound:
+                raise j.exceptions.Value(f"Your identity does not exists on {explorer_type} please setup a new identity with 3sdk")
+            if user.pubkey != j.me.encryptor.verify_key_hex:
+                raise j.exceptions.Value(f"Your identity does not match on {explorer_type} please create a new container with 3sdk instead")
+            j.me.tid = user.id
```

Comment:
no need to call `j.me.save()` here ?

Reply:
JSX> j.me._autosave                                                                                                                                                                  
False

I thought _autosave was enabled but it seems it's not, in all cases it's always better to not rely on it, ill add it